.github/workflows/codestyle.yml

```
@@ -51,3 +48,12 @@ jobs:
       - name: checkpatch --strict
         run: .github/workflows/checkpatch_list.sh ${CHK_CMD_OPTS}
                  --strict < PR_SHAs.txt
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # From time to time this will catch a git tag and change SOF_VERSION
+        with: {fetch-depth: 50, submodules: recursive}
+
+      - name: run yamllint
+        run: yamllint .github/workflows/*.yml
```

```suggestion
        run: yamllint --strict .github/workflows/*.yml
```
Just because we can, at least for now. See also below.